pyrasterframes/src/main/python/docs/raster-write.pymd

```diff
@@ -85,6 +84,19 @@ If there are many _tile_ or projected raster columns in the DataFrame, the GeoTI
 os.remove(outfile)
 ```
 
+## Overview Rasters
+
+In cases where writing and reading to/from a GeoTIFF isn't convenient, RasterFrames provides the `rf_agg_overview_raster` aggregate function, where you can construct a single raster (rendered as a tile) downsampled from all or a subset of the dataframe. This allows you to effectively construct the same operations the GeoTIFF writer performs, but without the file I/O.
```

Comment:
i think it is worth mentioning here that the user is going to specify the size of the resulting Tile.